simplefi-mstable-v2/src/vimAsset.ts

```diff
@@ -0,0 +1,151 @@
+import { Address, BigInt } from "@graphprotocol/graph-ts";
+import {
+  AccountLiquidityVIMAsset as AccountLiquidityVIMAssetEntity,
+  Market as MarketEntity,
+  VIMAsset as VIMAssetEntity
+} from "../generated/schema";
+import {
+  RewardAdded,
+  RewardPaid,
+  Staked,
+  VIMAsset,
+  Withdrawn
+} from "../generated/viMBTC/VIMAsset";
+import {
+  getOrCreateAccount,
+  investInMarket,
+  redeemFromMarket,
+  TokenBalance,
+  updateMarket
+} from "./common";
+
+function getRewardBalance(vimAssetAddress: Address, user: Address): BigInt {
+  let contract = VIMAsset.bind(vimAssetAddress)
+  let earned = contract.earned(user)
+  let unclaimed = contract.unclaimedRewards(user)
+  return earned.plus(unclaimed.value0)
```

Comment:
Should it be unclaimed rewards only that are counted as rewardBalance?

Reply:
done